sw/host/opentitanlib/src/util/openocd.rs

```diff
@@ -109,8 +109,8 @@ impl OpenOcdServer {
         let mut cmd = Command::new(&self.opts.openocd);
         cmd.args(args)
             .stdin(Stdio::null())
-            .stdout(Stdio::inherit())
-            .stderr(Stdio::inherit());
+            .stdout(Stdio::piped())
+            .stderr(Stdio::piped());
```

Comment:
Huh, I'm surprised that line 120 didn't panic, but I haven't dug into the docs.

Reply:
Yes it did, which is why I just PRed this. No one caught it because there are no tests in CI that use this openocd code yet (it's for manufacturing) and the initial PR was split into three and one change was lost in the process.

Reply:
And yes OpenOCD stdout/stderr will still appear, this just makes it nicer: every line appear prefixed by a timestamp and a `[openocd::stdout]` or `[openocd::stderr]`

Reply:
Got it. Still LGTM!